src/lime/_internal/backend/native/NativeCFFI.hx

```diff
@@ -1357,6 +1367,10 @@ class NativeCFFI
 
 	@:hlNative("lime", "hl_window_warp_mouse") private static function lime_window_warp_mouse(handle:CFFIPointer, x:Int, y:Int):Void {}
 
+	@:hlNative("lime", "hl_window_get_opacity") private static function lime_window_get_opacity(handle:CFFIPointer):Float;
```

Comment:
This function needs a body. It compiles the libs without it but a calling program cannot build. I changed it to :

	@:hlNative("lime", "hl_window_get_opacity") private static function lime_window_get_opacity(handle:CFFIPointer):Float {
		return 0;
	};
	
	For consistency with other functions I added an empty function body to set_opacity below too.